readme.md

```diff
@@ -124,7 +140,89 @@ There is no default export.
 
 Default schema ([`Schema`][api-schema]).
 
-Follows [GitHub][] style sanitation.
+Follows [GitHub][] style sanitation. 
+
+#### Overriding the default schema
+
+In some cases, it may be necessary to override the default schema.
+In the example below, the `id` attribute of the `<h2>` tag is removed 
+while the `<h1>` tag is unaltered.
+
+```js
+import rehypeParse from "rehype-parse"
+import rehypeSanitize from "rehype-sanitize"
+import rehypeStringify from "rehype-stringify"
+import { unified } from "unified"
+
+const file = await unified()
+  .use(rehypeParse)
+  .use(rehypeSanitize)
+  .use(rehypeStringify)
+  .process(
+    `<h1 id="bohemian">I'm just a poor boy</h1>
+<h2 id="rhapsody">From a poor family</h2>`
+  )
+
+console.log(String(file))
+
+// === Output ========
+// <h1 id="user-content-bohemian">I'm just a poor boy</h1>
+// <h2>From a poor family</h2>
```

Comment:
Hmm, if you don’t use this package directly here, perhaps it should go in `rehype-sanitize` instead?

Reply:
Also, this example doesn’t match project style. Save it in an `example.js`, run `npm test`, and you’d get it back formatted